pkg/encoding/kzgEncoder/multiframe.go

```diff
@@ -0,0 +1,221 @@
+package kzgEncoder
+
+import (
+	"bytes"
+	"encoding/gob"
+	"errors"
+	"fmt"
+
+	rs "github.com/Layr-Labs/eigenda/pkg/encoding/encoder"
+	kzg "github.com/Layr-Labs/eigenda/pkg/kzg"
+	bls "github.com/Layr-Labs/eigenda/pkg/kzg/bn254"
+)
+
+// Sample is the basic unit for a verification
+// A blob may contain multiple Samples
+type Sample struct {
+	Commitment bls.G1Point
+	Proof      bls.G1Point
+	RowIndex   int // corresponds to a row in the verification matrix
```

Comment:
Can you expand a bit mentioning that in our context, a row is a blob (so it's the same as blob index)?

Reply:
Every row in a matrix is a blob, row index tells which blob